server/templates/staff/course/assignment/assignment.new.html

```diff
@@ -25,33 +25,74 @@
 
     {% include 'alerts.html' %}
     <div class="row">
-      <div class="col-md-6">
+      <div class="col-md-12">
         <!-- Default box -->
           <div class="box">
             <div class="box-body">
-              {% call forms.render_form(form, action_url="", action_text='Create Assignment', class_='form') %}
-                  {{ forms.render_field(form.display_name, label_visible=true, placeholder='Hog', type='text') }}
-                  {{ forms.render_field(form.name, label_visible=true, value=current_course.offering + '/', placeholder='cal/cs61a/fa16/proj01', type='text') }}
+              {% call forms.render_form_bare(form, action_url="", class_='form new-assign-form wizard') %}
+                <h3>Name</h3>
+                <fieldset>
+                  {{ forms.render_field(form.display_name, label_visible=true, placeholder='Hog',
+                                        required="required", type='text', id="display-name") }}
+                  {{ forms.render_field(form.name, label_visible=true, value=current_course.offering + '/',
+                                        placeholder='cal/cs61a/fa16/proj01', type='text', id="assignment-offering") }}
                   {{ forms.render_field(form.max_group_size, label_visible=true, type='number', min='1', value='1') }}
-                  {{ forms.render_field(form.due_date, label_visible=true, value=utils.new_due_date(current_course), type='text', class='form-control datepicker') }}
-                  {{ forms.render_field(form.lock_date, label_visible=true, value=utils.new_lock_date(current_course), type='text', class='form-control datepicker') }}
-                  {{ forms.render_field(form.url, label_visible=true, placeholder='http://cs61a.org/proj/hog/ (Optional)', type='text') }}
-                  {{ forms.render_checkbox_field(form.visible, label_visible=true) }}
-                  {{ forms.render_field(form.autograding_key, label_visible=true, placeholder='5abc... (Optional)', type='text') }}
-                  {{ forms.render_checkbox_field(form.revisions_allowed, label_visible=true) }}
-                  {{ forms.render_checkbox_field(form.uploads_enabled, label_visible=true) }}
-                  {{ forms.render_field(form.upload_info, label_visible=true, placeholder='Upload the following files: test.py', type='text') }}
+                </fieldset>
+                <h3>Deadlines</h3>
+                <fieldset>
+                    {{ forms.render_field(form.due_date, label_visible=true, value=utils.new_due_date(current_course), type='text', class='form-control datepicker') }}
+                    {{ forms.render_field(form.lock_date, label_visible=true, value=utils.new_lock_date(current_course), type='text', class='form-control datepicker') }}
```

Comment:
It would be cool if this updated when the due date changes. Nice to have though, not necessary